File: source/two-sum.cpp
using namespace std;

class Solution {
public:
    vector<int> twoSum(vector<int>& nums, int target) {
        vector<int> solution = {0, 0};
        for(int i = 0; i<nums.size(); i++){
            for(int j = i+1; j<nums.size(); j++){
                if(nums.at(i)+nums.at(j) == target){
                    solution = {i, j};
                    return solution;
                }
            }
        }
        return solution;
    }
};

File: source/two-sum.py
class Solution:
    def twoSum(self, nums: List[int], target: int) -> List[int]:
        d = {}
        for i,num in enumerate(nums):
            
            if num in d.keys():
                return (i, d[num])
            else:
                d[target - num] = i
                
        return (0,0)
            
        
            
        